### PR TITLE
Release 8.1.2

### PR DESCRIPTION
**Pull request by markdboyd**

### CODE REVIEW

**Review by botanical — CHANGES_REQUESTED** [review]
I think you forgot to add the changes mentioned here https://github.com/nasa/cumulus/blob/master/CHANGELOG.md?plain=1#L175 👀

File: example/cumulus-tf/variables.tf [hunk 1]
```diff
@@ -441,3 +441,8 @@ variable "default_tags" {
   }
 }
 
+variable "cmr_custom_host" {
```
Comment (botanical) [inline example/cumulus-tf/variables.tf:444]: I think we can reference this in main.tf now

File: lambdas/sqs-message-remover/variables.tf [hunk 2]
```diff
@@ -23,7 +23,7 @@ variable "lambda_processing_role_arn" {
 # Optional
 
 variable "cmr_custom_host" {
-  description = "Custom host to use for CMR requests"
+  description = "Custom protocol and host to use for CMR requests (e.g. http://cmr-host.com)"
   type        = string
   default     = ""
```
Comment (botanical) [inline lambdas/sqs-message-remover/variables.tf:28]: default should be null, right?

File: tf-modules/archive/variables.tf [hunk 3]
```diff
@@ -147,7 +147,7 @@ variable "buckets" {
 }
 
 variable "cmr_custom_host" {
-  description = "Custom host to use for CMR requests"
+  description = "Custom protocol and host to use for CMR requests (e.g. http://cmr-host.com)"
   type        = string
   default     = ""
```
Comment (botanical) [inline tf-modules/archive/variables.tf:152]: Should this be null? Can we also reference this in `tf-modules/cumulus/archive.tf` now?

File: tf-modules/ingest/variables.tf [hunk 4]
```diff
@@ -17,7 +17,7 @@ variable "cmr_environment" {
 }
 
 variable "cmr_custom_host" {
-  description = "Custom host to use for CMR requests"
+  description = "Custom protocol and host to use for CMR requests (e.g. http://cmr-host.com)"
   type        = string
   default     = ""
```
Comment (botanical) [inline tf-modules/ingest/variables.tf:22]:
```suggestion
  default     = null
```

Comment (botanical) [reply tf-modules/ingest/variables.tf:22]: Is there a reason to not update `ingest.tf`?

File: lambdas/sqs-message-remover/variables.tf (hunk 2 above)
Comment (botanical) [reply lambdas/sqs-message-remover/variables.tf:28]: Is there a reason to not update `sqs-message-remover.tf`?

Comment (markdboyd) [reply lambdas/sqs-message-remover/variables.tf:28]: Hmm. That should have came in when I cherry-picked the commit. Good catch. Let me take a look.

File: example/cumulus-tf/variables.tf [hunk 5]
```diff
@@ -441,3 +441,14 @@ variable "default_tags" {
...
+  description = "Custom protocol and host to use for CMR requests (e.g. http://cmr-host.com)"
+  type        = string
+  default     = null
+}
+
+variable "cmr_custom_host" {
```
Comment (botanical) [inline example/cumulus-tf/variables.tf:450]: I think this is a duplicate

**Review by botanical — APPROVED** [review]
Made one comment about a duplicate variable but other than that, looks good

**Pull request merged**